health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/deliveryRule/AddDeliverycontext.js

```
@@ -554,36 +528,41 @@ const AddDeliveryRule = ({ targetedData, deliveryRules, setDeliveryRules, index,
+          onClose={() => {
+            setShowModal(false);
+          }}
+          footerChildren={[
+            <Button
+              // className={"campaign-type-alert-button"}
+              type={"button"}
+              size={"large"}
+              variation={"primary"}
+              label={t("CAMPAIGN_PRODUCTS_MODAL_SUBMIT_TEXT")}
+              onClick={confirmResources}
             />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)